map/style_tests/style_symbols_consistency_test.cpp

```diff
@@ -61,6 +62,58 @@ set<string> GetSymbolsSetFromResourcesFile(string const & density)
   return symbols;
 }
 
+template <typename TIterator, typename TFn>
+string Join(TIterator begin, TIterator end, TFn && fn)
```

Comment:
Why not? You may simply use transform iterators from boost and reuse JoinStrings.